file_converter/IFileWrapper.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace file_converter
+{
+    public interface IFileWrapper
+    {
+        string filePath { get; }
```

Comment:
Property names should be formatted in PascalCase. This one should be `FilePath`.